app/models/source.rb

```diff
@@ -9,7 +9,7 @@ class Source < ApplicationRecord
   # Service Catalog Inventory Objects
   has_many :service_credential_types
   has_many :service_credentials
-  has_many :service_offerings
+  has_many :service_offerings, :dependent => :destroy
```

Comment:
@hsong-rh Will this just mark the service_offering with archived_at and not delete it?

Reply:
@mkanoor I tested, it's deleted, not archived